api/models/users.js

```diff
@@ -15,9 +36,32 @@ class Users {
         return results;
     }
 
+    async delete(req) {
+        try{
+            await req.db.collection('users')
+            .remove({_id: Number(req.params.id) });
+        } catch (err ){
+            console.log('There is a problem when removing an user. Error: ' + err);
+        }
+       
+        return {results: 'Succeccfully removed'};
+    }
+
+    async update(req) {
+        try {
+            await req.db.collection('users')
+        .updateOne({_id: Number(req.params.id) },
+                   {$set: req.body});
+        } catch (err) {
+            console.log('There is a problem when updating an user. Error: ' + err);
+        }
+
+        return {results: 'Succeccfully updated'};
+    }
+
     async create(req) {
 
-        req.body._id = await this.getNextSequenceValue(req.db.collection('counter_users'));
+        req.query._id = await this.getNextSequenceValue(req.db.collection('counter_users'));
```

Comment:
As we are expecting a POST request in this case the object will come in the req.body, so we should keep the req.body._id instead of req.query._id for this function.

Reply:
@AdrianoMarra Okay. I changed it as req.body. It is ready to merge now.